src/components/Receive.jsx

```diff
@@ -109,6 +110,16 @@ export default function Receive({ currentWallet }) {
                     'Copy'
                   )}
                 </rb.Button>
+                <input
+                  aria-hidden
+                  ref={inputRef}
+                  defaultValue={address}
```

Comment:
```suggestion
                  value={address}
```
Just noticed, that when you click "Get new address", the value of the input changes, however, it will only ever copy the first address. Using `value` instead of `defaultValue` seems to fix it.